wdl-lint/tests/lints/disallowed-declaration-name/source.wdl

```diff
@@ -0,0 +1,55 @@
+version 1.2
+
+#@ except: MissingRequirements, SnakeCase
```

Comment:
```suggestion
#@ except: MissingRequirements, SnakeCase, InputSorting, NonmatchingOutput
```
`source.errors` should only have diagnostics relevant to what is being tested. Add these exceptions and rebless